### PR TITLE
Development

### PR DESCRIPTION
**Pull request by Asetre**
make a pr next time like this

### CODE REVIEW

**Comment by Asetre** [comment]
Merge it

File: server.js [hunk 1]
```diff
@@ -14,8 +14,7 @@ app.use(bodyParser.json());
 mongoose.Promise = global.Promise;
 
 const db = require('./config/keys').mongoURI;
-
-const port = process.env.PORT || 5000;
+const port = require('./config/keys').port;
```
Comment (Asetre) [inline server.js]: const { port } = require('./config/keys')

File: routes/api/users.js [hunk 2]
```diff
@@ -1,43 +1,35 @@
+'use strict'
 const express = require('express');
 const router = express.Router();
-const gravatar = require('gravatar');
 const bcrypt = require('bcryptjs');
 
-const User = require('../../models/Users')
+const { User } = require('../../models/Users')
```
Comment (Asetre) [inline routes/api/users.js]: export default User?
const User = require('....')

Comment (gedfaj) [reply routes/api/users.js]: I don't think export default works in node js.

**Pull request merged**